modules/gitops-configuring-groups-and-argocd-rbac.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * configuring-sso-for-argo-cd-on-openshift.adoc
+
+[id="configuring-groups-and-argocd-rbac_{context}"]
+= Configure groups and Argo CD RBAC
+
+Role-based access control (RBAC) allows you to provide relevant permissions to users.
+
+.Prerequisites 
+
+* You have created the `ArgoCDAdmins` group in Keycloak.
+
+* The user you want to give permissions to has logged in to Argo CD.
+ 
+.Procedure
+
+. In the Keycloak dashboard navigate to *Users* -> *Groups*. Add the user to the Keycloak group `ArgoCDAdmins`. 
+
+. Ensure that `ArgoCDAdmins` group has the required permissions in the `argocd-rbac` ConfigMap. 
```

Comment:
Use config map throughout this module as per: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#general-references